ui/apps/platform/src/Containers/VulnMgmt/List/ImageComponents/VulnMgmtListImageComponents.js

```diff
@@ -50,7 +50,7 @@ export function getComponentTableColumns(workflowState) {
             sortField: componentSortFields.COMPONENT,
         },
         {
-            Header: `CVEs`,
+            Header: `Image CVEs`,
```

Comment:
I think we need to feature flag the name, the entityType line, and the link building logic below ,for this.

Reply:
Ah yes, I will do that. Thanks